packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -41,6 +45,17 @@ export abstract class TableColumn<
     @attr({ attribute: 'sorting-disabled', mode: 'boolean' })
     public sortingDisabled = false;
 
+    /** @internal */
+    @observable
+    public isValidContentAndHasOverflow = false;
+
+    /** @internal */
+    public headerSpan!: HTMLSpanElement;
+
+    /** @internal */
+    @observable
+    public contentSlot?: HTMLSlotElement;
```

Comment:
How are you deciding when to use the non-null assertion and when to say it's nullable?
Both of these are created by the same template, not sure why their nullability is treated different

Reply:
Make sure and clean-up side-effects of the change. i.e. can remove the optional chaining from contentSlot reference.